settings/FixedDueDateScheduleForm.js

```diff
@@ -143,28 +154,34 @@ Delete
   }
 
   renderPaneTitle() {
-    const { initialValues, stripes: { intl } } = this.props;
+    const {
+      initialValues,
```

Comment:
InitialValues direcly reassigned after destructuring. So, it make sence to reassign in the destructuring and  also set default value